vue-app/src/store/index.ts

```diff
@@ -183,8 +183,12 @@ export const mutations = {
       state.recipient[payload.step] = payload.updatedData[payload.step]
     }
   },
-  [TOGGLE_SHOW_CART_PANEL](state) {
-    state.showCartPanel = !state.showCartPanel
+  [TOGGLE_SHOW_CART_PANEL](state, isOpen: boolean | undefined) {
+    if (isOpen) {
```

Comment:
This is going to have a weird behavior. If `isOpen` is false, then you will end up toggling it instead of closing it.
Perhaps, we could check not for falsy values but for undefined values.
```
if (typeof isOpen === 'undefined') {
  state.showCartPanel = !state.showCartPanel
} else {
  state.showCartPanel = isOpen
}
```

Reply:
Added the case of checking for null and undefined in conditional. Thanks for the suggestion.